th2_data_services_lwdp/provider_api/grpc.py

```diff
@@ -78,15 +78,7 @@ def search_events(
         self,
         start_timestamp: int = None,
         end_timestamp: int = None,
-        parent_event: str = None,
-        search_direction: str = "NEXT",
-        resume_from_id: str = None,
-        result_count_limit: int = None,
-        keep_open: bool = False,
-        limit_for_parent: int = None,
-        metadata_only: bool = True,
-        attached_messages: bool = False,
-        filters: Optional[List[Filter]] = None,
+        parent_event: str = None
```

Comment:
Please sure that LwDP supports this parameter

Reply:
It does.